sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -6084,4 +6084,15 @@ CREATE TABLE labmessage (
 CREATE TABLE labmessage_history (LIKE labmessage);
 
 INSERT INTO schema_version (version_number, comment) VALUES (291, 'Add LabMessage #3486');
+-- SurvNet Adaptations - Create new field “Salutation” for persons #3411
+ALTER TABLE person
+    ADD COLUMN salutation varchar(255),
+    ADD COLUMN othersalutation varchar(512);
```

Comment:
Just confirmed with Maté, we should make this text instead of varchar(512), as text has a variable size.

Reply:
It is wired, there was a ticket when all text columns were changed to fixed size varchar, we don't have any text column in our DB. Why would we switch now to text?

Reply:
ok, I talked with Mate. I will change the varchar to text